src/main/java/Duke/Commands/CommandHelp.java

```diff
@@ -0,0 +1,13 @@
+package Duke.Commands;
+
+import Duke.DukeException.DukeException;
+import Duke.System.Storage;
+import Duke.System.TaskList;
+import Duke.System.Ui;
+
+public class CommandHelp extends Command {
```

Comment:
Did not use the constructor in this case? I guess you could have used a function instead of making this a class